dev-python/holoviews/holoviews-1.14.2.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+# HoloViews imports from "distutils" at runtime.
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="Make data analysis and visualization seamless and simple"
+HOMEPAGE="
+	https://holoviews.org
+	https://pypi.org/project/holoviews
+	https://github.com/holoviz/holoviews"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+BDEPEND=">=dev-python/setuptools-30.3.0[${PYTHON_USEDEP}]"
```

Comment:
We can drop this and IUSE as ```DISTUTILS_USE_SETUPTOOLS=rdepend``` [includes](https://dev.gentoo.org/~mgorny/python-guide/distutils.html#distutils-use-setuptools) BDEPEND.

(This isn't true for the RDEPEND variable in general, it's just what the eclass does.)

Reply:
You got it. :+1: